Projects/Server/Network/Packets/IncomingAccountPackets.cs

```diff
@@ -379,15 +519,17 @@ public static void GameLogin(NetState state, CircularBufferReader reader, int pa
             state.Disconnect("Unable to find auth id.");
         }
 
-        if (state._authId != 0 && authID != state._authId || state._authId == 0 && authID != state._seed)
+        state.Version = ap.Version;
+
+        if (state._authId != 0 && authID != state._authId || state._authId == 0 && authID != state._seed && !state.IsKRClient) // Temp fix, excludes KR, valid for no crypt client.
```

Comment:
Add check here for SA too.

Reply:
I just double checked this, this change is not required by EC, auth/seed seems to be fully compatible to 2D.